server/data/mockData/orderSummary.ts

```diff
@@ -2,7 +2,7 @@ import { OrderSummary } from '../../interfaces/orderSummary'
 
 const orderSummary: OrderSummary = {
   keyOrderDetails: {
-    legacySubjectId: '1234567',
+    legacySubjectID: '1234567',
```

Comment:
A pedantic one. I feel it should be `legacySubjectId` but defer to @JazJax on this.

Reply:
Agree - "Id" tends to result in better readability - e.g. legacySubjectIdMethod is more readable than legacySubjectIDMethod. Hence, I would always prefer 'Id' over 'ID' for consistency.